tethysapp/data_rods_explorer/public/js/load_pages.js

```diff
@@ -12,7 +12,7 @@ function onClickLink(link, navItem) {
     } else {
         $('.nav-link').removeClass('open');
         $(link).addClass('open');
-        $('.nav-item').addClass('hidden');
+        //$('.nav-item').addClass('hidden');
         $('#nav-' + navItem).removeClass('hidden');
```

Comment:
Why is line 15 commented out? `'hidden'` probably should be changed to `'d-none'` here.

Reply:
I don't recall commenting that out, nor see any reason it should be.  Uncommenting and changing to d-none